homework/index.js

```diff
@@ -29,25 +32,150 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  // 1- create function to add row
+  function createRow(table, label, content) {
+    const row = createAndAppend('tr', table);
+    createAndAppend('th', row, { text: label });
+    createAndAppend('td', row, { text: content });
+    return row;
+  }
+
+  // 2- create details of repositories
+  function renderRepoDetails(detailsSection, repo) {
+    const li = createAndAppend('table', detailsSection, { class: 'item' });
```

Comment:
Your variable name `li` suggests that you are creating an `<li>` element, whereas in truth you are erronously creating a `<table>` element. As you are also creating a `<table>` element in line 64, the resulting HTML is invalid:

```html
<section class="left-section">
  <table class="item">
    <table>
      <tr>
        <!-- etc -->
      </tr>
    </table>
  </table>
</section>
```